bpm/bonita-api/bonita-common-api/src/main/java/org/bonitasoft/engine/bpm/process/ModelFinderVisitor.java

```diff
@@ -45,65 +45,65 @@
 /**
  * author Emmanuel Duchastenier
  */
-public interface ModelFinderVisitor<T> {
+public interface ModelFinderVisitor {
```

Comment:
I think that since the purpose of ModelFinderVisitor is not anymore to find an element (it's the ExpressionFinder that find expression) but only to visit the model we can rename ModelFinderVisitor to ModelVisitor and find method to visit